### ISSUE TITLE
List out of range error when running inference on complex

### ISSUE THREAD

**Issue by carze** [aqlaboratory/openfold#170]
Hi, 

I've been attempting to get OpenFold (main branch) running on a multiFASTA to produce a complex prediction but seem to keep running into the following error:

```
Traceback (most recent call last):
  File "run_pretrained_openfold.py", line 592, in <module>
    main(args)
  File "run_pretrained_openfold.py", line 406, in main
    args,
  File "run_pretrained_openfold.py", line 227, in generate_feature_dict
    fasta_path=tmp_fasta_path, super_alignment_dir=alignment_dir,
  File "/home/ubuntu/tools/openfold/openfold/openfold/data/data_pipeline.py", line 829, in process_multiseq_fasta
    template_features = unify_template_features(template_feature_list)
  File "/home/ubuntu/tools/openfold/openfold/openfold/data/data_pipeline.py", line 112, in unify_template_features
    k: np.concatenate([od[k] for od in out_dicts]) for k in out_dicts[0]
IndexError: list index out of range
```
My input sequences look like this:

```
>1a
MPWWYRRRSYNPWRRRNWFRRPRKTIYRRYRRRRRWVRRKPFYKRKIKRLNIVEWQPKSI
RKCRIKGMLCLFQTTEDRLSYNFDMYEESIIPEKLPGGGGFSIKNISLYALYQEHIHAHN
IFTHTNTDRPLARYTGCSLKFYQSKDIDYVVTYSTSLPLRSSMGMYNSMQPSIHLMQQNK
LIVPSKQTQKRRKPYIKKHISPPTQMKSQWYFQHNIANIPLLMIRTTALTLDNYYIGSRQ
LSTNVTIHTLNTTYIQNRDWGDRNKTYYCQTLGTQRYFLYGTHSTAQNINDIKLQELIPL
TNTQDYVQGFDWTEKDKHNITTYKEFLTKGAGNPFHAEWITAQNPVIHTANSPTQIEQIY
TASTTTFQNKKLTDLPTPGYIFITPTVSLRYNPYKDLAERNKCYFVRSKINAHGWDPEQH
QELINSDLPQWLLLFGYPDYIKRTQNFALVDTNYILVDHCPYTNPEKTPFIPLSTSFIEG
RSPYSPSDTHEPDEEDQNRWYPCYQYQQESINSICLSGPGTPKIPKGITAEAKVKYSFNF
KWGGDLPPMSTITNPTDQPTYVVPNNFNETTSLQNPTTRPEHFLYSFDERRGQLTEKATK
RLLKDWETKETSLLSTEYRFAEPTQTQAPQEDPSSEEEEESNLFERLLRQRTKQLQLKRR
IIQTLKDLQKLE
>1a
MPWWYRRRSYNPWRRRNWFRRPRKTIYRRYRRRRRWVRRKPFYKRKIKRLNIVEWQPKSI
RKCRIKGMLCLFQTTEDRLSYNFDMYEESIIPEKLPGGGGFSIKNISLYALYQEHIHAHN
IFTHTNTDRPLARYTGCSLKFYQSKDIDYVVTYSTSLPLRSSMGMYNSMQPSIHLMQQNK
LIVPSKQTQKRRKPYIKKHISPPTQMKSQWYFQHNIANIPLLMIRTTALTLDNYYIGSRQ
LSTNVTIHTLNTTYIQNRDWGDRNKTYYCQTLGTQRYFLYGTHSTAQNINDIKLQELIPL
TNTQDYVQGFDWTEKDKHNITTYKEFLTKGAGNPFHAEWITAQNPVIHTANSPTQIEQIY
TASTTTFQNKKLTDLPTPGYIFITPTVSLRYNPYKDLAERNKCYFVRSKINAHGWDPEQH
QELINSDLPQWLLLFGYPDYIKRTQNFALVDTNYILVDHCPYTNPEKTPFIPLSTSFIEG
RSPYSPSDTHEPDEEDQNRWYPCYQYQQESINSICLSGPGTPKIPKGITAEAKVKYSFNF
KWGGDLPPMSTITNPTDQPTYVVPNNFNETTSLQNPTTRPEHFLYSFDERRGQLTEKATK
RLLKDWETKETSLLSTEYRFAEPTQTQAPQEDPSSEEEEESNLFERLLRQRTKQLQLKRR
IIQTLKDLQKLE
>1a
MPWWYRRRSYNPWRRRNWFRRPRKTIYRRYRRRRRWVRRKPFYKRKIKRLNIVEWQPKSI
RKCRIKGMLCLFQTTEDRLSYNFDMYEESIIPEKLPGGGGFSIKNISLYALYQEHIHAHN
IFTHTNTDRPLARYTGCSLKFYQSKDIDYVVTYSTSLPLRSSMGMYNSMQPSIHLMQQNK
LIVPSKQTQKRRKPYIKKHISPPTQMKSQWYFQHNIANIPLLMIRTTALTLDNYYIGSRQ
LSTNVTIHTLNTTYIQNRDWGDRNKTYYCQTLGTQRYFLYGTHSTAQNINDIKLQELIPL
TNTQDYVQGFDWTEKDKHNITTYKEFLTKGAGNPFHAEWITAQNPVIHTANSPTQIEQIY
TASTTTFQNKKLTDLPTPGYIFITPTVSLRYNPYKDLAERNKCYFVRSKINAHGWDPEQH
QELINSDLPQWLLLFGYPDYIKRTQNFALVDTNYILVDHCPYTNPEKTPFIPLSTSFIEG
RSPYSPSDTHEPDEEDQNRWYPCYQYQQESINSICLSGPGTPKIPKGITAEAKVKYSFNF
KWGGDLPPMSTITNPTDQPTYVVPNNFNETTSLQNPTTRPEHFLYSFDERRGQLTEKATK
RLLKDWETKETSLLSTEYRFAEPTQTQAPQEDPSSEEEEESNLFERLLRQRTKQLQLKRR
IIQTLKDLQKLE
```

I wasn't sure if it this was an issue with using the same sequence identifier for each sequence but even changing them to be unique (producing 3 alignments) I still run into the same error. Am I doing something else wrong here?

Thanks!

**Comment by gahdritz** [comment]
I pushed a fix in 236c686. Try it out and let me know how it goes.

**Comment by carze** [comment]
Thanks for the quick fix, this did the trick!
